src/pages/AppDetail.svelte

```diff
@@ -40,20 +53,39 @@
     item={app}
     schema={APPLICATION_DEFINITION_SCHEMA} />
 
-  <TabGroup active="Metrics">
-    <Tab key="Metrics">Metrics</Tab>
-    <Tab key="Pings">Pings</Tab>
-    <Tab key="Application IDs">Application IDs</Tab>
+  <TabGroup
+    active={itemType}
+    on:tabChanged={({ detail }) => {
+      itemType = detail.active;
+      updateURL();
+    }}>
+    <Tab key="metrics">Metrics</Tab>
+    <Tab key="pings">Pings</Tab>
+    <Tab key="app_ids">Application IDs</Tab>
 
-    <TabContent key="Pings">
-      <ItemList itemType="pings" items={app.pings} appName={app.app_name} />
+    <TabContent key="pings">
+      <ItemList
+        itemType="pings"
+        items={app.pings}
+        appName={app.app_name}
+        filterText={queryString}
+        on:filterTextChanged={({ detail }) => {
+          updateURL(detail.filterText);
```

Comment:
We need to be careful to pass this value up when updating the URL-- if we use the local `queryString` parameter, it will be out of date.

Reply:
This turns out not to be required if we use a binding!